### PR TITLE
Add support for MISR SOM projection

### PR DESCRIPTION
**Pull request by mbull2016**
This adds a new projection type 'misrsom' with '+path=' argument for MISR path numbers.  It is similar to existing 'lsat' projection type.

### CODE REVIEW

**Comment by hobu** [comment]
Thanks for the contribution, but this pull request fails both compilation tests. It will have to be working and tested before we can merge it.

**Comment by mbull2016** [comment]
I'm not sure how to fix the automatic compilation test.  I get a successful build on Fedora 20 with the following:

```
sh autogen.sh
./configure CFLAGS=-std=c99 
make
```

**Comment by mbull2016** [comment]
Compilation error fixed

File: html/man_cs2cs.html [hunk 1]
```diff
@@ -1,6 +1,9 @@
-<HTML><HEAD><TITLE>Manpage of PROJ</TITLE>
+Content-type: text/html; charset=UTF-8
```
Comment (QuLogic) [inline html/man_cs2cs.html:1]: This shouldn't be here, I think.

**Comment by mbull2016** [comment]
Closing to start again with a fresh branch

**Pull request closed without merging**